Automattic/wp-calypso
# A4A: Add Sidebar navigation component

This pull request adds a new sidebar component for the A4A portal. The component is based on the Jetpack cloud and has been modified to fit the A4A use cases. It's worth noting that the background color will be updated once the brand color is added to the Color Studio.

<img width="276" alt="Screenshot 2024-02-15 at 7 40 33 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/37094dd2-e884-4ec9-8aad-9acbb72052cf">
<img width="274" alt="Screenshot 2024-02-15 at 7 40 44 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/1f0c6c4f-fb68-4b07-84ec-ac1751c92aad">



Closes https://github.com/Automattic/jetpack-genesis/issues/222

## Proposed Changes

* Implement an A4A Sidebar component based on the Jetpack cloud sidebar. Most of the codes are copied from the source. 
* Add sample pages to demonstrate the new sidebar.

## Testing Instructions

* Checkout to this branch.
* Spin up the environment using the command yarn `start-a8c-for-agencies`.
* Goto http://agencies.localhost:3000/
* Confirm that the sidebar shows the logo and user menu in the footer.
* Click the **Sites** menu item to go to the Sites page
* Confirm that the Site selector and user menu are visible in the header and the 'Get help' link is visible in the footer.


**NOTE: Note that I didn't register /overview and /sites URL so please always start at `/`.**

## Pre-merge Checklist
